src/components/layout/Navbar/style.module.scss

```diff
@@ -48,6 +52,10 @@
       gap: 1rem;
       line-height: 24px;
       margin-right: auto;
+
+      &.hidden {
```

Comment:
Instead of adding `.hidden` to every element on the navbar, it might be easier to track the state of the whole navbar and use a single selector on the entire navbar to select everything at once that you need to change.

first line is bad psuedo-code
```
.header[mobile] {
  .iconLink {
    display: none;
  }
  .portalLinks {
    display: none;
  }
  etc
}
```